src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -1111,16 +1112,21 @@ void Connection::close_due_to_protocol_error(std::error_code ec, std::optional<s
 void Connection::close_due_to_client_side_error(std::error_code ec, std::optional<std::string_view> msg,
```

Comment:
I guess this (std::error_code) will be removed/addressed in later PR's

Reply:
Yeah. I'm just trying to do this in stages.